### DISCUSSION TITLE
Translations

### DISCUSSION THREAD

**Discussion by BetaZavr** [community/community#37347] [Other Feature Feedback, Questions, & Ideas]
Good day.
How can you be contacted to send translations into various languages of the world?
(17 lang files)

**Comment by ossanna16** [comment]
Hi @BetaZavr,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on translations. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
